runner/daemon/src/mill/daemon/MillBuildBootstrap.scala

```diff
@@ -76,15 +76,18 @@ class MillBuildBootstrap(
     }
 
     Watching.Result(
-      watched = runnerState.frames.flatMap(f => f.evalWatched ++ f.moduleWatched),
+      watched = runnerState.watched,
       error = runnerState.errorOpt,
       result = runnerState
     )
   }
 
-  def evaluateRec(depth: Int): RunnerState = {
-    // println(s"+evaluateRec($depth) " + recRoot(projectRoot, depth))
+  def evaluateRec(
+      depth: Int,
+      closeEvaluator: Boolean = true
```

Comment:
Does this need to be individually configured for every `evaluateRec` call? If not we should pass it in the constructor instead

Reply:
My bad, this is code from former changes, that have been dropped. Just removed this parameter.